packages/perseus/src/widgets/interactive-graphs/graphs/utils.ts

```diff
@@ -42,10 +42,7 @@ export const getIntersectionOfRayWithBox = (
             return [xAtYExtreme, yExtreme];
 
         default:
-            // This default case is only reachable if the input is invalid
-            // (initialPoint is outside the graph bounds, or initialPoint and
-            // throughPoint are the same).
-            return [0, 0];
+            return [xExtreme, yExtreme];
```

Comment:
I don't know why codecov says this line isn't covered. It definitely is covered (by the test I added). If I remove it, the test fails.